Open Judge System/OJS.Common/BackgroundJobs/HangfireBackgroundJobService.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace OJS.Common.BackgroundJobs
+{
+    using System;
+    using System.Linq.Expressions;
+
+    using global::Hangfire;
+    using OJS.Common.BackgroundJobs.Contracts;
+
+    public class HangfireBackgroundJobService : IBackgroundJobService
+    {
+        public void AddOrUpdateRecurringJob(object recurringJobId, Expression<Action> methodCall, string cronExpression)
+            => RecurringJob.AddOrUpdate((string)recurringJobId, methodCall, cronExpression);
```

Comment:
`=>` on the upper line, if its more than 120 characters, separate parameters on new lines.